probe-rs/src/probe/wlink/mod.rs

```diff
@@ -379,6 +379,14 @@ impl DebugProbe for WchLink {
 
 /// Wrap WCH-Link's USB based DMI access as a fake JTAGAccess
 impl JTAGAccess for WchLink {
+    fn reset_jtag_state_machine(&mut self) -> Result<(), DebugProbeError> {
+        Ok(())
```

Comment:
Shouldn't this be a `NotImplemented`?

Reply:
Wouldn't that cause issues? WchLink is a weird probe that isn't actually jtag, I think it should be okay to just ignore very jtag-specific calls.

Reply:
Marked as resolved since the trait method no longer exists in JTAGAccess 🙃

I guess if it's not relevant for smart probes, we only have to need it implemented in the bit-level code.